### PR TITLE
[#1160] Update New York (Colony) to New York (State)

### LINKED ISSUE #1160: Update New York (Colony) to New York (State)
[This file](https://digitalhub.northwestern.edu/files/0e5436bb-270e-4320-860c-d61834cead23) incorrectly had the geographic term "New York (State)" map to "New York (Colony)". Update the memoized file to correct this and check for other potentially incorrect mappings.

### PR DESCRIPTION
**Pull request by Meowcenary**
Update memoized LCNAF, memoized MeSH, and memoized LCSH file to include terms that were searched for in the latest export and terms that were corrected. closes https://github.com/galterlibrary/digital-repository/issues/1160

### CODE REVIEW

**Review by mattbaumann1 — APPROVED** [review]

**Comment by Meowcenary** [comment]
This is going to be on hold for a bit. There are some general updates to the LCNAF terms that need to be made. I'm going to also check out the LCSH and MeSH terms since it's likely something is off there too.

**Comment by fenekku** [comment]
Will wait for further changes. Thanks!

**Review by mattbaumann1 — APPROVED** [review]

File: memoized_lcnaf.json [hunk 1]
```diff
@@ -347,5 +347,14 @@
   "Singapore Subject": "https://id.loc.gov/authorities/names/n79059023",
   "Singapore": "https://id.loc.gov/authorities/names/n79059023",
   "United States. Army.": "https://id.loc.gov/authorities/names/n79063202",
-  "United States. Army": "https://id.loc.gov/authorities/names/n79063202"
+  "United States. Army": "https://id.loc.gov/authorities/names/n79063202",
+  "New York (State)": "https://id.loc.gov/authorities/names/n80126293",
+  "Belarus": "https://id.loc.gov/authorities/names/n91128701.html",
```
Comment (fenekku) [inline memoized_lcnaf.json:352]: Let's remove the `.html` suffix. The no suffix form redirects to the .html one. The no suffix form is the correct identifier form we use.

File: memoized_lcsh.json [hunk 2]
```diff
@@ -410,5 +410,52 @@
   "Viral Proteins": "https://id.loc.gov/authorities/subjects/sh94006884",
   "Cytology": "https://id.loc.gov/authorities/subjects/sh99005436",
   "Lincoln Park (Chicago, Ill.)": "https://id.loc.gov/authorities/subjects/sh89006458",
-  "Piazza dei Miracoli (Pisa, Italy)": "https://id.loc.gov/authorities/subjects/sh2008002257"
+  "Piazza dei Miracoli (Pisa, Italy)": "https://id.loc.gov/authorities/subjects/sh2008002257",
+  "Skin--Inflammation": "http://id.loc.gov/authorities/subjects/sh85123181",
```
Comment (fenekku) [inline memoized_lcsh.json:414]: All of these should be with the `https` form e.g. 

```suggestion
  "Skin--Inflammation": "https://id.loc.gov/authorities/subjects/sh85123181",
```

Review terms below to use https all the time.

File: memoized_lcsh.json [hunk 3]
```diff
@@ -1,414 +1,461 @@
 {
-  "Semantic Web": "http://id.loc.gov/authorities/subjects/sh2002000569",
-  "Abdomen--Cancer": "http://id.loc.gov/authorities/subjects/sh85000095",
-  "Cancer": "http://id.loc.gov/authorities/subjects/sh85019492",
+  "Semantic Web": "https://id.loc.gov/authorities/subjects/sh2002000569",
```
Comment (Meowcenary) [inline memoized_lcsh.json:2]: Updated file to consistently use HTTPS

File: memoized_lcsh.json [hunk 4]
```diff
@@ -1,414 +1,461 @@
...
+  "Diaphragm": "https://id.loc.gov/authorities/subjects/sh85037585",
+  "Cardiovascular system--Diseases": "https://id.loc.gov/authorities/subjects/sh85020229",
+  "Metformin": "https://id.loc.gov/authorities/subjects/sh2007006278",
+  "Overweight persons": "https://id.loc.gov/authorities/subjects/sh89004326",
+  "Cognition disorders": "https://id.loc.gov/authorities/subjects/sh85027745",
+  "Alzheimer's disease--Patients--Care": "https://id.loc.gov/authorities/subjects/sh91003917",
+  "Dementia--Patients--Care": "https://id.loc.gov/authorities/subjects/sh2008102144",
+  "Contact tracing (Epidemiology)": "https://id.loc.gov/authorities/subjects/sh2020006779",
+  "Parkinson's disease--Research": "absent",
+  "Parkinson's disease": "https://id.loc.gov/authorities/subjects/sh85098115.html",
```
Comment (fenekku) [inline memoized_lcsh.json:433]: remove html suffix

**Pull request merged**